trinity/protocol/bcc/proto.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+from p2p.protocol import Protocol
+
+from trinity.protocol.bcc.commands import (
+    Status,
+    GetBeaconBlocks,
+    BeaconBlocks,
+    AttestationRecords,
+)
+
+
+class BCCProtocol(Protocol):
+    name = "bcc"
+    version = 0
+    _commands = [Status, GetBeaconBlocks, BeaconBlocks, AttestationRecords]
+    cmd_length = 4
```

Comment:
Semi-confident :)

If I understand correctly, the `cmd_length` is only used for protocols higher up the stack so that they can choose command ids that don't have collisions with lower level protocols (by offsetting them with the sum of all `cmd_length`s below). I assume `eth` and `les` use a larger number than necessary to reserve some commands for future use.